### DISCUSSION TITLE
In README.md for 22.x, a part of the Table of Contents is transcribed on line 746.

### DISCUSSION THREAD

**Discussion by tadashi-oya** [telekom-security/tpotce#1065] [General]
In README.md for 22.x, a part of the Table of Contents is transcribed on line 746.
If I were to write it, it seems to be better to start from line 674.
https://github.com/telekom-security/tpotce/blob/555bf2245dfed17cea026cad6bc6128aca19a63b/README.md?plain=1#L746-L755

And in the 22.x README.md, there is what appears to be a typo on line 73.
https://github.com/telekom-security/tpotce/blob/555bf2245dfed17cea026cad6bc6128aca19a63b/README.md?plain=1#L73

**Comment by t3chn0m4g3** [comment]
Thanks for spotting it! Simply forgot to put it back.
Fixed in https://github.com/telekom-security/tpotce/commit/bfbb77a5a06bd3d0b4983a50f59f7c5dd5162d31
